nxcomp/Loop.cpp

```diff
@@ -13954,19 +13021,15 @@ void PrintProcessInfo()
          << strTimestamp() << "'.\n";
   }
 
-  #ifdef TEST
 
   if (*errorsFileName != '\0')
   {
-    cerr << "Info" << ": Using errors file '" << errorsFileName << "'.\n";
-  }
+    cerr << "Info" << ": Using errors file '" << errorsFileName << "'.\n";  }
 
   if (*statsFileName != '\0')
   {
-    cerr << "Info" << ": Using stats file '" << statsFileName << "'.\n";
-  }
+    cerr << "Info" << ": Using stats file '" << statsFileName << "'.\n";  }
```

Comment:
The last two changes only remove correct newlines.

Because it's outputting directly to cerr, we should keep these lines in mind though. More on that later.